tools/yac_clipper/yac.xml

```diff
@@ -41,7 +41,7 @@
     <outputs>
         <data format="fasta" metadata_source="input" name="output" label="Clipping of ${input.name}">
           <change_format>
-              <when input="out_format" value="fastq" format="fastq" />
+              <when input="out_format" value="fastq" format="fastqsanger" />
```

Comment:
wait, we can turn this around ... make the default format_source, and then in the when statement `<when input="out_format" value="fasta" format="fasta"`